user.go

```diff
@@ -116,12 +125,59 @@ func (s *UserService) GetSelf() (*User, *Response, error) {
 	return &user, resp, nil
 }
 
+// WithMaxResults sets the max results to return
+func WithMaxResults(maxResults int) userSearchF {
+	return func(s userSearch) userSearch {
+		s = append(s, userSearchParam{name: "maxResults", value: fmt.Sprintf("%d", maxResults)})
+		return s
+	}
+}
+
+// WithStartAt set the start pager
+func WithStartAt(startAt int) userSearchF {
+	return func(s userSearch) userSearch {
+		s = append(s, userSearchParam{name: "startAt", value: fmt.Sprintf("%d", startAt)})
+		return s
+	}
+}
+
+// WithActive sets the active users lookup
+func WithActive(active bool) userSearchF {
+	return func(s userSearch) userSearch {
+		s = append(s, userSearchParam{name: "includeActive", value: fmt.Sprintf("%t", active)})
+		return s
+	}
+}
+
+// WithInactive sets the inactive users lookup
+func WithInactive(inactive bool) userSearchF {
+	return func(s userSearch) userSearch {
+		s = append(s, userSearchParam{name: "includeInactive", value: fmt.Sprintf("%t", inactive)})
+		return s
+	}
+}
+
 // Find searches for user info from JIRA:
 // It can find users by email, username or name
 //
 // JIRA API docs: https://docs.atlassian.com/jira/REST/cloud/#api/2/user-findUsers
-func (s *UserService) Find(property string) ([]User, *Response, error) {
-	apiEndpoint := fmt.Sprintf("/rest/api/2/user/search?username=%s", property)
+func (s *UserService) Find(property string, tweaks ...userSearchF) ([]User, *Response, error) {
```

Comment:
Will changing the function signature affect code that was written before the change?

Reply:
Nope, as you can see in the original test for this functionality. The API is backwards compatible.

Reply:
Perfect!